### ISSUE TITLE
This is does not work

### ISSUE THREAD

**Issue by swood** [roundcube/roundcubemail-docker#111]
Hi.

I try to use your compose file docker-compose-fpm-alpine.yaml. It has created three containers, but it doesn't work.
The container with database wrote to log:
```
2020-11-19 00:20:40.389 UTC [1] LOG:  database system is ready to accept connections
2020-11-19 00:20:45.053 UTC [28] FATAL:  password authentication failed for user "roundcube"
2020-11-19 00:20:45.053 UTC [28] DETAIL:  Password does not match for user "roundcube".
        Connection matched pg_hba.conf line 99: "host all all all md5"
```

After that, the container roundcubemail wrote:
```
Please check the 'mime_types' config option and run this script again.
Executing database schema update.
ERROR: SQLSTATE[08006] [7] FATAL:  password authentication failed for user "roundcube"
ERROR: Failed to connect to database
All done.
Loading composer repositories with package information
```
which is fair, because the database on the previous step did not initialised.

My question is, why this with this unacceptable status and how you would suggest using these images?

**Comment by robin-thoni** [comment]
The problem sounds obvious:
```
password authentication failed for user "roundcube"
Password does not match for user "roundcube"
```

You either, have the wrong password on roundcube, or the database didn't update the user password on first launch. If you used an existing database, or updated the password in the environment after the first database run, this is expected.

**Comment by swood** [comment]
Hi @robin-thoni 
Neither of that. I have created new containers from scratch via compose file. Nothing else.

**Comment by thomascube** [comment]
Might be related to #112 when docker compose uses a shared volume. In this case any changes to `ROUNDCUBEMAIL_DB_PASSWORD` are not applied unless you destroy the volume.

**Comment by bremby** [comment]
Hi,
the docker-compose file doesn't work for me either. I tried both the file from the `examples` folder and the one from the `fpm` folder. The former seems more complete, as the log also contains messages from the `roundcubemail` container, while the latter doesn't. The result is the same either way, though: nginx serves only the "Welcome to nginx" page.

I have tried deleting the volumes and existing containers, nothing helps. I'm also not sure whether I *have to* provide a custom nginx config or whether those environment variables are enough. I'm more experienced with k8s manifests, so maybe I'm missing something trivial.

Interestingly, the log from roundcubemail container shows:
```
roundcubemail     | WARNING: /var/www/html/config/config.inc.php already exists.
roundcubemail     | ROUNDCUBEMAIL_* environment variables have been ignored.
```

Any help appreciated. Thanks. :)
